tests/test_env.py

```diff
@@ -75,6 +76,7 @@ def test_create_isolated_build_has_with_pip(tmp_path, capfd, mocker):
 
 @pytest.mark.skipif(sys.version_info[0] == 2, reason='venv module used on Python 3 only')
 def test_fail_to_get_script_path(mocker):
+    sysconfig.get_config_vars()  # the config vars are globally cached and may use get_path, make sure they are created
```

Comment:
I don't really like that we need to remember to put this any time we do the `get_path` mocking.

A few options to handle it:

1. Create an autouse session- or module-scoped fixture that does `sysconfig.get_config_vars()` so that the cache is always warmed.
2. Create a fixture for a `get_path_mocker` that warms the cache, like this:

```python
@pytest.fixture
def get_path_mocker(mocker):
    import sysconfig
    # This is a global cache, and may use get_path, so make sure to call it before mocking
    sysconfig.get_config_vars()
    def patch(side_effect=None):
        mocker.patch('sysconfig.get_path', side_effect=side_effect)
    yield patch
```

There are other ways to do this as well. Dealer's choice.

Reply:
Yeah can make it an auto use session fixture.